packages/grafana-ui/src/utils/storybook/withStoryContainer.tsx

```diff
@@ -44,38 +43,8 @@ const StoryContainer: React.FC<{ width?: number; height?: number; showBoundaries
 };
 
 export const withStoryContainer = (story: RenderFunction) => {
-  const CONTAINER_GROUP = 'Container options';
-  // ---
-  const containerBoundary = boolean('Show container boundary', false, CONTAINER_GROUP);
-  const fullWidthContainer = boolean('Full width container', false, CONTAINER_GROUP);
-  const containerWidth = number(
-    'Container width',
-    300,
-    {
-      range: true,
-      min: 100,
-      max: 500,
-      step: 10,
-    },
-    CONTAINER_GROUP
-  );
-  const containerHeight = number(
-    'Container height',
-    0,
-    {
-      range: true,
-      min: 100,
-      max: 500,
-      step: 10,
-    },
-    CONTAINER_GROUP
-  );
   return (
-    <StoryContainer
-      width={fullWidthContainer ? undefined : containerWidth}
-      height={containerHeight}
-      showBoundaries={containerBoundary}
-    >
+    <StoryContainer width={300} height={0} showBoundaries={false}>
```

Comment:
I don't know how the clean up plan for Storybook looks like so this question might be out of place, sorry if that is the case. With this change you're removing the possibility for the user of Storybook to resize the story. Not sure this is an important feature for us but we should check if this is needed. 

Is there a way to use Storybook controls to achieve the same resize feature? I can see that the decorators now have a second argument passed to them which is the StoryContext. Can this be used somehow?

```ts
export const withStoryContainer = (story: RenderFunction, context: StoryContext) => {
  return (
    <StoryContainer width={300} height={0} showBoundaries={false}>
      {story()}
    </StoryContainer>
  );
};
```

![image](https://user-images.githubusercontent.com/562238/122167462-4773c200-ce7b-11eb-96c5-934ba6ab6f29.png)

Reply:
Great catch! Would investigate and see how we could use the StoryContext here.